src/components/Checkbox/Checkbox.st.css

```diff
@@ -0,0 +1,95 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultTextColor: color-5;
+    DefaultBorderColor: color-5;
+    DefaultIconColor: color-8;
+    DisabledColor: color-3;
+    Font: font("{theme: 'Body-M', size: '16px', lineHeight: '1.5em'}");
+}
+
+:vars {
+    /*Overrides*/
+    TextColor: --overridable;
+    BorderColor: --overridable;
+    IconColor: color-8;
+}
+
+:vars {
+    CurrentBorderColor: color(fallback(value(BorderColor), value(DefaultBorderColor)));
+    CurrentIconColor: color(fallback(value(IconColor), value(DefaultIconColor)));
+    CurrentTextColor: color(fallback(value(TextColor), value(DefaultTextColor)));
+}
+
+.root {
+    -st-states: checked, disabled, error;
+
```

Comment:
We should probably add
```
:import {
  -st-from: "wix-ui-core/index.st.css";
  -st-named: Checkbox;
}

.root {
  -st-extends: Checkbox; 
}
```
here
So that changes made in `wix-ui-core` will be reflected here

Reply:
@jonathanadler done